### PR TITLE
docs(nextjs): add nextjs webpack 5 guide

### PR DESCRIPTION
**Pull request by kirjai**
Added a small guide describing how to enable webpack 5 for next.js applications

## Related Issue(s)

Fixes #

### CODE REVIEW

File: nx-dev/nx-dev/public/documentation/latest/react/guides/nextjs-webpack5.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Enable webpack 5 for Next.js
+
+As of Nx version 12.4.0 - users can can opt in to use webpack 5 for building and serving Next.js applications.
```
Comment (ild0tt0re) [inline nx-dev/nx-dev/public/documentation/latest/react/guides/nextjs-webpack5.md:3]: Typo error: two times **can**

**Comment by nnennajohn** [comment]
In latest Nextjs version, webpack5 is enabled by default. So wondering if this doc change is still relevant. Also, it actually currently does not work with NX. So following this PR for when it is fixed: h[ttps://github.com/nrwl/nx/pull/5892](https://github.com/nrwl/nx/pull/5892)

**Comment by kirjai** [comment]
@nnennajohn we're still working on support for Next.js 11, so for now, you have to manually enable it in `next.config.js`

> Also, it actually currently does not work with NX

hm, using Nx 12.4.0 with Next.js 10.2.0 should have webpack 5 working, once you enable it as per the docs in this PR. does it not work for you?

**Review by brandonroberts — APPROVED** [review]
LGTM

**Comment by nnennajohn** [comment]
@kirjai I did try installing Next 11 and updating to webpack 5, but that did not work. So perhaps it's still relevant for Next 10. Did not try at Next 10. 

Waiting on the PR I reference earlier before trying again. 

Thank you.

**Pull request merged**